dev-docs/adunit-reference.md

```diff
@@ -261,7 +290,7 @@ pbjs.addAdUnits({
 
 <a name="adUnit-video-example-outstream"></a>
 
-#### Outstream
+#### No Content/Standalone
```

Comment:
How about "Standalone ad with no video content"?

Reply:
Same thing here. I'm trying to match the exact IAB definitions to avoid confusion:
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo

Reply:
## Standalone ad with no video content
This is the scenario formerly known as Outstream. As of __DATE__, the IAB now calls this scenario "No Content/Standalone".